packages/web-components/fast-foundation/src/carousel/carousel.template.ts

```diff
@@ -0,0 +1,103 @@
+
+const playIcon: string = `<svg viewBox="0 0 16 16" width="16px" height="16px" xmlns="http://www.w3.org/2000/svg"><path d="M1 0V16L15.0083 8L1 0Z"></path></svg>`;
+
+const basicTemplate: ViewTemplate = html<Carousel>`
+    <div class="carousel-content" ${ref("basicContent")}>
+        <slot ${slotted({ property: "items", filter: elements() })}></slot>
+    </div>
+`;
+
+const tabbedTemplate: ViewTemplate = html<Carousel>`
+    <div class="carousel-content">
+        <div class="tablist" part="tablist" role="tablist" ${ref("tablistRef")}>
+            <slot class="tab" name="tab" part="tab" ${slotted("tabs")}></slot>
+        </div>
+        <div class="tabpanel" ${ref("tabPanelsContainerRef")}>
+            <slot name="tabpanel" part="tabpanel" ${slotted("tabpanels")}></slot>
+        </div>
+    </div>
+`;
+
+/**
+ * The template for the {@link @microsoft/fast-foundation#(Carousel:class)} component.
+ * @public
+ */
+export const CarouselTemplate = html<Carousel>`
+<template
+    role="group"
+    aria-roledescription="carousel"
+>
+    <div
+        class="rotation-control-container"
+        part="rotation-control-container"
+        aria-label="${x =>
+            x.paused
+                ? "start automatic slide rotation"
+                : "stop automatic slide rotation"}"
+        @keypress=${(x, c) => x.handleRotationKeyDown(c.event as KeyboardEvent)}
+        ${ref("rotationControlContainer")}
+    >
+        <slot
+            name="rotation-control"
+            part="rotation-control"
+        >
+            <fast-button appearance="neutral" ${ref("rotationControlDefault")}>
```

Comment:
Since this is a scenario where we use other fast components internally, we need to consider that in the fluent world the prefix will be different. We established a basic pattern for that. @scomea can update you on what we're doing as a solution. Then we just need to update all the elements used inside of this template to use that.